docs/getting_started/faq.rst

```diff
@@ -0,0 +1,20 @@
+Frequently Asked Questions
+==========================
+
+.. _faq1:
+
+Q: How can I refresh a comment/subreddit/submission?
+A: .. code-block:: python
+
+reddit.comment(id=comment.id)
+reddit.subreddit(display_name=subreddit.display_name)
+reddit.submission(id=submission.id)
+
+.. _faq2:
+
+Q: Whenever I try to do anything, I get an ``invalid_grant`` error
+A: This means that the account you are trying to sign in with has 2FA enabled,
```

Comment:
Is this the only reason that `invalid_grant` occurs?

Reply:
So far it's the main reason. I would assume an invalid password could also be it.

Reply:
I think we should mention that, if invalid password will also result in it. E.g., something like, this typically means either your password is incorrect, of you have 2FA enabled and you haven't provided the correct 2FA code.